app/CZIcompress.cpp

```diff
@@ -95,13 +95,15 @@ int main(int argc, char** argv)
 
     // Create an "output-stream-object"
     const auto output_stream =
-        libCZI::CreateOutputStreamForFile(utils::utf8::WidenUtf8(command_line_options.GetOutputFileName()).c_str(), false);
+        libCZI::CreateOutputStreamForFile(utils::utf8::WidenUtf8(command_line_options.GetOutputFileName()).c_str(), command_line_options.GetOverwriteExistingFile());
 
-    // create the "CZI-writer"-object
-    const auto writer = libCZI::CreateCZIWriter();
+    // create (and configure) the "CZI-writer"-object
+    libCZI::CZIWriterOptions czi_writer_options;
+    czi_writer_options.allow_duplicate_subblocks = command_line_options.GetIgnoreDuplicateSubblocks();
+    const auto writer = libCZI::CreateCZIWriter(&czi_writer_options);
 
     // GUID_NULL here means that a new Guid is created
-    const auto czi_writer_info = std::make_shared<libCZI::CCziWriterInfo>(GUID{0, 0, 0, {0, 0, 0, 0, 0, 0, 0, 0}});
+    const auto czi_writer_info = std::make_shared<libCZI::CCziWriterInfo>(libCZI::GUID{0, 0, 0, {0, 0, 0, 0, 0, 0, 0, 0}});
```

Comment:
Could we define a libCZI::GUID() default constructor or method which returns a new GUID rather than defining it as null?

Reply:
Agreed, this looks a bit clunky. And yes, I guess we could do something about it (since the last refactoring, the [GUID-struct](https://github.com/ptahmose/libczi-zeiss/blob/31bf4ccd8b125bbe01b28c072990ba5c6201330b/Src/libCZI/libCZI_Utilities.h#L24C1-L90C7) now is a C++-only struct and lives inside the libCZI-namespace).
Let's leave this for some later PR then :-)